lago/cmd.py

```diff
@@ -398,6 +400,44 @@ def do_console(prefix, host, **kwargs):
 
 
 @lago.plugins.cli.cli_plugin(
+    help='Create Ansible hosts file of the environment'
+)
+@in_lago_prefix
+@with_logging
+def do_generate_ansible_hosts(prefix, out_format, **kwargs):
+    """
+    This method iterate through all the VMs and create an ansible
+    hosts configuration. It defines an IP address and private key
+    for the machine in group named by its type, it is printed to
+    standard output. The content looks for example like this:
+
+    [ovirt-host]
+    1.2.3.4 ansible_host=1.2.3.4 ansible_ssh_private_key_file=/path/id_rsa
+    1.2.3.6 ansible_host=1.2.3.6 ansible_ssh_private_key_file=/path/id_rsa
+    [ovirt-engine]
+    1.2.3.5 ansible_host=1.2.3.5 ansible_ssh_private_key_file=/path/id_rsa
+    """
```

Comment:
This doesn't format nicely in the CLI :/
```
positional arguments:
  VERB
    status
    copy-from-vm        Copy file from a virtual machine to local machine
    shell               Open shell on the domain or run as script/command
    console             Open serial console to the domain
    deploy
    copy-to-vm          Copy file/dir to a virtual machine from the local host
    revert              Revert resources to a snapshot
    ansible_hosts       This method iterate through all the VMs and create an
                        ansible hosts configuration. It defines an IP address
                        and private key for the machine in group named by its
                        type, it is printed to standard output. The content
                        looks for example like this: [ovirt-host]
                        192.168.201.2 ansible_host=192.168.201.2
                        ansible_ssh_private_key_file=/path/id_rsa
                        192.168.201.4 ansible_host=192.168.201.4
                        ansible_ssh_private_key_file=/path/id_rsa [ovirt-
                        engine] 192.168.201.3 ansible_host=192.168.201.3
                        ansible_ssh_private_key_file=/path/id_rsa
```
Also - I'm not sure about the VM types as keys, if not using ovirtlago, they would all be ``default``, maybe using the name then would be better then(if there is no specific type).